src/Servers/Kestrel/Core/src/Middleware/ListenOptionsConnectionLoggingExtensions.cs

```diff
@@ -31,7 +31,8 @@ public static ListenOptions UseConnectionLogging(this ListenOptions listenOption
         {
             var loggerFactory = listenOptions.KestrelServerOptions.ApplicationServices.GetRequiredService<ILoggerFactory>();
             var logger = loggerName == null ? loggerFactory.CreateLogger<LoggingConnectionMiddleware>() : loggerFactory.CreateLogger(loggerName);
-            listenOptions.Use(next => new LoggingConnectionMiddleware(next, logger).OnConnectionAsync);
+            var enableWiresharkStyleLog = listenOptions.KestrelServerOptions.EnableWiresharkStyleLog;
```

Comment:
We should use a new overload on UseConnectionLogging rather than a new property on KestrelServerOptions to enable this.

The new overload could take a enableWiresharkStyleLog bool directly. Or we could create a ConnectionLoggingOptions object. Thoughts @Tratcher @davidfowl?

P.S. I haven't brought this up in triage yet. Probably shouldn't have marked it as a "good first issue" until we got consensus we should do this. Hopefully everyone else agrees this is a good idea.

Reply:
I prefer `ConnectionLoggingOptions`